packages/editor/src/components/post-actions/export-pattern-action.js

```diff
@@ -23,9 +24,9 @@ function getJsonFromItem( item ) {
 	return JSON.stringify(
 		{
 			__file: item.type,
-			title: item.title || item.name,
-			content: item.patternPost.content.raw,
-			syncStatus: item.patternPost.wp_pattern_sync_status,
+			title: getItemTitle( item ),
+			content: item.content.raw,
```

Comment:
In the DataViews, pattern content is indeed present in `item.content.raw`. But when exported from the sidebar, the content is present in `item.content`, so there seems to be no content `field` in the exported JSON file.

However, pattern export from the sidebar doesn't seem to have worked before. See [this comment](https://github.com/WordPress/gutenberg/issues/63217#issuecomment-2212320713) for more details.

Reply:
I have a small PR here:

- https://github.com/WordPress/gutenberg/pull/63227